migrations/m190819_210853_initial.php

```diff
@@ -16,7 +16,7 @@ public function safeUp()
             'id' => $this->primaryKey(),
             'token' => $this->string(255),
             'user_id' => $this->integer()->notNull(),
-            'created_at' => $this->dateTime()->defaultValue(new \yii\db\Expression('NOW()')),
```

Comment:
I'm not sure this is valid here. Maybe we should leave the Expression here.

Reply:
Why? If only because this is a migration, then why did you approve similar changes in another modules like this https://github.com/humhub/humhub-modules-custom-pages/pull/146/files?
Ah, I even find same question here https://github.com/humhub/humhub-modules-fcm-push/pull/7 without answer from you.
Please explain me so I will know for future, either just confirm to revert this change, thanks.

Reply:
In this case it is the default value of the table field. I would either set no default value here or continue to use DB Expression (NOW). The current solution would always set the time of migration as default value. 

The other approved changes, were not a default value but really an actual value of an entry (insert).

Reply:
ok, I reverted the default value.